tests/admin_views/models.py

```diff
@@ -929,3 +929,14 @@ class ExplicitlyProvidedPK(models.Model):
 
 class ImplicitlyGeneratedPK(models.Model):
     name = models.IntegerField(unique=True)
+
+
+# Model for #12952
```

Comment:
Is it possible to reuse some existing models by adding `verbose_name` to them? That's usually preferred since adding models adds complexity and slows the test suite.